agents/gce/fence_gce.py

```diff
@@ -399,14 +438,57 @@ def define_new_opts():
 		"required" : "0",
 		"order" : 12
 	}
+	all_opt["earlyexit"] = {
+		"getopt" : "",
+		"longopt" : "earlyexit",
+		"help" : "--earlyexit                    Return early if reset is already in progress",
+		"shortdesc" : "If an existing reset operation is detected, the fence agent will return before the operation completes with a 0 return code.",
+		"required" : "0",
+		"order" : 13
+	}
+	all_opt["warntimeout"] = {
```

Comment:
You can use "second" as type for the timeouts.

Reply:
Done